lib/puma/dsl.rb

```diff
@@ -65,7 +66,10 @@ def self.ssl_bind_str(host, port, opts)
         v_flags = (ary = opts[:verification_flags]) ?
           "&verification_flags=#{Array(ary).join ','}" : nil
 
-        "ssl://#{host}:#{port}?cert=#{opts[:cert]}&key=#{opts[:key]}" \
+        cert_flags = (cert = opts[:cert]) ? "cert=#{Puma::Util.escape(opts[:cert])}" : nil
+        key_flags = (cert = opts[:key]) ? "&key=#{Puma::Util.escape(opts[:key])}" : nil
```

Comment:
No need for the `cert` assignment here, right?

Reply:
Oops, thanks: https://github.com/puma/puma/pull/2857